lib/portage/dbapi/bintree.py

```diff
@@ -1234,11 +1234,32 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "true")
```

Comment:
Instead of executing "true" when PORTAGE_TRUST_HELPER is unset, we should just return early.

Reply:
Please resolve that with @thesamesam ...

Reply:
Done